benchmark/misc/freelist.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var common = require('../common.js');
+var FreeList = require('../../lib/internal/freelist').FreeList;
+
+var bench = common.createBenchmark(main, {
+  n: [100000]
+});
+
+function main(conf) {
+  var n = conf.n;
+  var poolSize = 500;
```

Comment:
It might be better to at least use the freelist max size currently used in core by the http module: 1000.